### DISCUSSION TITLE
Artisan v2.8.0 released

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#993] [Announcement]
With added support for the between batch protocol and the addition of a phases widget to the Comparator the [Artisan v2.8.0 release](https://artisan-roasterscope.blogspot.com/2022/10/artisan-v280.html) focus on repeatability and consistency in roasting.

**Comment by nestormcy** [comment]
hello thanks for all your work, I am a manufacturer of coffee roasters in Venezuela and I would like to know if you could compile the Artisan application for the aarch64 architecture for the new orange pi development boards and thus all the machines that you manufacture will have that system I would appreciate a contribution.

**Reply by MAKOMO** [reply]
Artisan is open-source and its GPLv3 licence does not allow to incorporate it into your machines. If you do anyhow, you will get into legal troubles!

We have not any plans to support the aarch64 platform.
